linker/spm.eu0.lst

```diff
@@ -7,6 +7,26 @@
 
 /* Dol */
 
+800e658c:evt_env_blur_on
+
+// iValues.h
+80df32d0:finalRpgChild
+80df1b70:underchompAttack1
+80df1eb0:underchompAttack2
+80df21f0:underchompAttack3
+80df2530:checkWinOrContinue
+80def500:attacking
+80def9d8:usePixls
+80df0500:useItems
+80df1958:runningAway
+80def438:techChild1
+80def3ec:techChild2
+80DF334C:theParentOfBeginRPG
+80c6cb08:techtext1
+80c6f2ec:rpgTribePatch1
+80c6d618:rpgTribePatch2
+80c6d624:underchompRepeatPatch
```

Comment:
Like we talked about on discord, mods shouldn't be putting specific instructions in lsts, just using offsets from actual symbols